firefly_client/firefly_client.py

```diff
@@ -175,7 +113,7 @@ def _handle_event(self, ev):
     #    'dataType': ['STRING', 'JSON', 'BG_STATUS'],
     #    'data': {'channel': , 'connID': }
     # }
-    def received_message(self, m):
+    def _received_message(self, m):
```

Comment:
It might be confusing to users to see, so maybe the underscore should be removed?